src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -829,6 +829,78 @@ class ExposureSubmissionCoordinator: NSObject, RequiresAppDependencies {
 		push(topBottomContainerViewController)
 	}
 
+	// MARK: TestCertificateInfo
+
+	private func showTestCertificateInfo() {
+		let testCertificateInfoViewController = ExposureSubmissionTestCertificateInfoViewController(
+			ExposureSubmissionTestCertificateViewModel(
```

Comment:
```suggestion
		let testCertificateViewController = ExposureSubmissionTestCertificateViewController(
			ExposureSubmissionTestCertificateViewModel(
```

I guess with the `ExposureSubmission` prefix its precise enough, we don't need `Info` in the name (Already missing from the view model 😁)